### PR TITLE
Dirble: update license and version numbering

### PR DESCRIPTION
**Pull request by sciguy16**
I've updated the pkgbuild for dirble to match its versioning scheme and license

### CODE REVIEW

**Comment by Edu4rdSHL** [comment]
I don't understand why we're copying files when we can juts run `cargo build --release` and then use `install -Dm 755 target/release/$pkgname "$pkgdir/usr/bin/$pkgname"`

Any specific reason?

Also, if you want to use a minor `pkgver` number, you need to add `epoch`, see https://wiki.archlinux.org/index.php/PKGBUILD#epoch

File: packages/dirble/PKGBUILD [hunk 1]
```diff
@@ -2,21 +2,21 @@
 # See COPYING for license details.
 
 pkgname=dirble
-pkgver=186.c1b7237
+pkgver=1.4.0
```
Comment (Edu4rdSHL) [inline packages/dirble/PKGBUILD]: Add epoch.

File: packages/dirble/PKGBUILD [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
 groups=('blackarch' 'blackarch-webapp' 'blackarch-scanner')
 arch=('x86_64' 'armv6h' 'armv7h' 'aarch64')
 url='https://github.com/nccgroup/dirble'
-license=('MIT')
+license=('GPL3')
 makedepends=('git' 'cargo' 'rust')
 source=('git+https://github.com/nccgroup/dirble.git')
 sha512sums=('SKIP')
 
 pkgver() {
```
Comment (Edu4rdSHL) [inline packages/dirble/PKGBUILD]: Remove the whole pkgver() function if you're going to use versioned model.

File: packages/dirble/PKGBUILD [hunk 3]
```diff
@@ -2,21 +2,21 @@
...
-pkgver=186.c1b7237
+pkgver=1.4.0
 pkgrel=1
 pkgdesc='Fast directory scanning and scraping tool.'
 groups=('blackarch' 'blackarch-webapp' 'blackarch-scanner')
 arch=('x86_64' 'armv6h' 'armv7h' 'aarch64')
 url='https://github.com/nccgroup/dirble'
-license=('MIT')
+license=('GPL3')
 makedepends=('git' 'cargo' 'rust')
```
Comment (Edu4rdSHL) [inline packages/dirble/PKGBUILD]: Remove git if you're going to use versioned model.

File: packages/dirble/PKGBUILD [hunk 4]
```diff
@@ -2,21 +2,21 @@
...
 source=('git+https://github.com/nccgroup/dirble.git')
```
Comment (Edu4rdSHL) [inline packages/dirble/PKGBUILD]: Change the source pointing to the release version (like `$pkgname::https://github.com/nccgroup/dirble/archive/v$pkgver.tar.gz`)

**Comment by ikstream** [comment]
I think it was on purpose, that we use with git `pkgver` instead of releases

File: packages/dirble/PKGBUILD [hunk 5]
```diff
@@ -2,21 +2,21 @@
...
 sha512sums=('SKIP')
 
 pkgver() {
   cd $pkgname
 
-  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+  awk -F'"' '/^version/ {print $2}' Cargo.toml
 }
 
 build() {
```
Comment (Edu4rdSHL) [inline packages/dirble/PKGBUILD]: Use `cargo build --release` instead of make.

**Comment by sciguy16** [comment]
Thanks for the pointers, I've updated it to fully commit to versioned releases. It only needs cargo as a make dependency, because cargo implies rust. And namcap pointed out that libcurl is a runtime dependency, so I've added that too.

**Pull request merged**